controller/user.js

```diff
@@ -5,8 +5,10 @@ const db = require("../models/index.js");
 const models = db.sequelize.models;
 const jsonwebtoken = require("jsonwebtoken");
 const errors = require("../utils/error");
+const bcrypt = require("bcryptjs");
+const { getHashedPassword } = require("./auth.js");
 
-function buildUserJson(user) {
+function formatUser(user) {
```

Comment:
For `/api/v1/signup`, email is included
For `/api/v1/users/:user_name`, email is not included

Suggestion: change the function signature to:

```
function formatUser(user, publicFieldsOnly=false)
```

When `publicFieldsOnly` is true, exclude `email` from the result.